### ISSUE TITLE
Germ cell arrest

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9819]
**Preferred term label:**
Germ cell arrest

**Synonyms**

**Definition (free text, please give PubMed ID)**
This term summarises all types of arrest, including the two types of spermatogenesis arrest (spermatogonial and spermatocyte arrest) as well as spermatogenesis arrest (round spermatid arrest). 

**Parent term (use hpo.jax.org/app)**
HP:0011961

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399771

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr  The term label needs to be comprehensible outside of the HPO hierarchy. Can we change the label to 
**Male germ cell arrest**
or something like that?

**Comment by MagoWyr** [comment]
Yes, "Male germ cell arrest" is fine
________________________________

**Comment by pnrobinson** [comment]
Also, "This term summarises all types of arrest" is not a good definition since it presupposes that the reader knows what arrest is. Please provide a definition that can be understood by a PhD student or a physician from another discipline (or let me know if I should draft definitions).

**Comment by MagoWyr** [comment]
Okay, here is a definition for this term:

Germ cell arrest labels all types of arrest in the process of forming haploid spermatozoa, including arrests during spermatogenesis (spermatogonial arrest and spermatocyte arrest) as well as during spermiogenesis (round spermatid arrest).
________________________________

**Comment by pnrobinson** [comment]
Looking again, I think that the intention of the current term [Spermatogenesis maturation arrest HP:0031038](https://hpo.jax.org/app/browse/term/HP:0031038) is to group all forms of male germ cell arrest.

This term does seem to be used in the community, e.g., 
https://en.wikipedia.org/wiki/Spermatogenesis_arrest

There is no sense in making a new term if it only has one child -- then in essence, they are synonyms
https://github.com/obophenotype/human-phenotype-ontology/issues/9821

**Comment by pnrobinson** [comment]
added/finalized per discussion
